### DISCUSSION TITLE
Hiding future events from the What's New page?

### DISCUSSION THREAD

**Discussion by OldGuyInTheClub** [TeamNewPipe/NewPipe#10706] [Questions]
Asking here vs. filing an issue since it is a UI/usage question vs. a bug or a feature.  Newpipe is working fine.

Config: v026.1, Samsung Galaxy S10+, Android 12

I subscribe to a channel that has Live sessions that they schedule into the future, resulting in placeholders for these events.  When I update my What's New page, the three top results are for their next three sessions, currently one, two, and four weeks out.  This is nice on one hand to know what's coming up but it makes Feed Updates harder to find since everything else is pushed down.

Is it possible to suppress or hide these future events?

Steps I've tried:
- Marking the future vids as "Watched" via long press.  No effect after refreshing.  Future sessions still appear
- Settings -> Content -> Fetch Channel Tabs and _unchecking_ Tracks, Shorts, and Live.  No effect after refreshing. Future sessions still appear
- Checked other Settings pages.  Don't see anything applicable
- Searched Issues and Discussions.  Didn't find anything with search terms I tried

Thank you.

OGitC

**Comment by opusforlife2** [comment]
There's an eye icon on the main page. Tap on it and uncheck Upcoming.

**Reply by OldGuyInTheClub** [reply]
Thank you, very much.
